<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1030</width>
    <height>599</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">*{border:none}</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="fr_left">
      <property name="minimumSize">
       <size>
        <width>240</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>240</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">*{border: none}

QFrame{
background-color: #22222C;
}

QPushButton {
    color: #8e8f93;
    font-family: &quot;Calibri&quot;;
    font-size: 10pt;
    font-weight: bold;   
    line-height: 29px;
    letter-spacing: 1px;
    text-align: left;
}

QPushButton:hover{
	color: #f2f2f2;
    text-align: left;    
}


QPushButton:hover#bt_0{
    icon: url(:/img/icon_light/home.svg);
}

QPushButton:hover#bt_1{
    icon: url(:/img/icon_light/book.svg);
}

QPushButton:hover#bt_2{
    icon: url(:/img/icon_light/bar-chart-2.svg);
}

QPushButton:hover#bt_3{
    icon: url(:/img/icon_light/settings.svg);
}

QPushButton:hover#bt_4{
    icon: url(:/img/icon_light/bell.svg);
}

QPushButton:pressed{
	color: #8e8f93;
    text-align: left;    
}

QPushButton:pressed#bt_0{
    icon: url(:/img/icon_gray/home.svg);
}

QPushButton:pressed#bt_1{
    icon: url(:/img/icon_gray/book.svg);
}

QPushButton:pressed#bt_2{
    icon: url(:/img/icon_gray/bar-chart-2.svg);
}

QPushButton:pressed#bt_3{
    icon: url(:/img/icon_gray/settings.svg);
}

QPushButton:pressed#bt_4{
    icon: url(:/img/icon_gray/bell.svg);
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>5</number>
       </property>
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>30</number>
       </property>
       <item>
        <widget class="QFrame" name="fr_side">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <property name="horizontalSpacing">
           <number>65</number>
          </property>
          <property name="verticalSpacing">
           <number>25</number>
          </property>
          <property name="leftMargin">
           <number>30</number>
          </property>
          <property name="topMargin">
           <number>50</number>
          </property>
          <property name="rightMargin">
           <number>15</number>
          </property>
          <property name="bottomMargin">
           <number>60</number>
          </property>
          <item row="0" column="0">
           <widget class="QPushButton" name="bt_0">
            <property name="text">
             <string>   Go home</string>
            </property>
            <property name="icon">
             <iconset resource="icon.qrc">
              <normaloff>:/img/icon_gray/home.svg</normaloff>:/img/icon_gray/home.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>14</width>
              <height>14</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="bt_1">
            <property name="text">
             <string>   Register</string>
            </property>
            <property name="icon">
             <iconset resource="icon.qrc">
              <normaloff>:/img/icon_gray/book.svg</normaloff>:/img/icon_gray/book.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>14</width>
              <height>14</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="bt_2">
            <property name="text">
             <string>   Analytics</string>
            </property>
            <property name="icon">
             <iconset resource="icon.qrc">
              <normaloff>:/img/icon_gray/bar-chart-2.svg</normaloff>:/img/icon_gray/bar-chart-2.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>14</width>
              <height>14</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QPushButton" name="bt_4">
            <property name="text">
             <string>   Updates</string>
            </property>
            <property name="icon">
             <iconset resource="icon.qrc">
              <normaloff>:/img/icon_gray/bell.svg</normaloff>:/img/icon_gray/bell.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>14</width>
              <height>14</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLabel" name="label_3">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel{
	color: #f2f2f2;
    font-size: 10pt;
    letter-spacing: 0px;
    font-family: &quot;Roboto&quot;;
    font-weight: Normal;   
    text-align: center;
	border-radius: 8px;
	background-color: #7672FB;
}</string>
            </property>
            <property name="text">
             <string> 17</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QPushButton" name="bt_3">
            <property name="text">
             <string>   Settings</string>
            </property>
            <property name="icon">
             <iconset resource="icon.qrc">
              <normaloff>:/img/icon_gray/settings.svg</normaloff>:/img/icon_gray/settings.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>14</width>
              <height>14</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>379</height>
          </size>
         </property>
        </spacer>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="icon.qrc">
           <normaloff>:/img/icon_light/star.svg</normaloff>:/img/icon_light/star.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>35</width>
           <height>35</height>
          </size>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label">
         <property name="styleSheet">
          <string notr="true">QLabel{
	color: #f2f2f2;
    font-size: 13pt;
    letter-spacing: 1px;
    font-family: &quot;Calibri&quot;;
    font-weight: Bold;   
    text-align: center;
}</string>
         </property>
         <property name="text">
          <string>Premium</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label_2">
         <property name="styleSheet">
          <string notr="true">QLabel{
    color: #8e8f93;
    font-family: &quot;Calibri&quot;;
    font-size: 10pt;
    font-weight: Normal;   
    line-height: 29px;
    letter-spacing: 0px;
    text-align: left;
}</string>
         </property>
         <property name="text">
          <string>Will give you everthing you need 
For your projects as a developer. 
</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QPushButton" name="bt_5">
         <property name="minimumSize">
          <size>
           <width>185</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	color: #f2f2f2;
    font-size: 10pt;
    letter-spacing: 1px;
    font-family: &quot;Open Sans&quot;;
    font-weight: bold;   
    text-align: center;
	border-radius: 20px;
	background-color: qlineargradient(spread:pad, x1:0.2, y1:0, x2:1, y2:0.8, stop:0 rgba(122, 119, 			240, 255), stop:0.8 rgba(164, 122, 229, 255), stop:1 rgba(171, 113, 169, 255));
}


QPushButton:hover {
	background-color: rgb(165, 115, 237);
}


QPushButton:pressed{
    border: 1px solid  ;
	background-color: rgb(165, 115, 237);
}

</string>
         </property>
         <property name="text">
          <string>Upgrade</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #2e2e3c;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item alignment="Qt::AlignRight|Qt::AlignTop">
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QCheckBox" name="checkBox_2">
            <property name="styleSheet">
             <string notr="true">QCheckBox::indicator:unchecked {
    image: url(&quot;:/img/icon_gray/padlock.svg&quot;);
    width: 16px; /* Ancho del icono */
    height: 16px; /* Alto del icono */
}

QCheckBox::indicator:checked {
    image: url(&quot;:/img/icon_light/lock-open.svg&quot;);
    width: 16px; /* Ancho del icono */
    height: 16px; /* Alto del icono */
}
</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="styleSheet">
             <string notr="true">QLabel{
    color: #f2f2f2;
    font-family: &quot;Roboto&quot;;
    font-size: 10pt;
    font-weight: Normal;   
    line-height: 29px;
    letter-spacing: 2px;
    text-align: left;
}</string>
            </property>
            <property name="text">
             <string>Aditional Tools</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox">
            <property name="styleSheet">
             <string notr="true">QCheckBox::indicator:unchecked {
    image: url(&quot;:/img/icon_gray/toggle-button-left.png&quot;);
    width: 40px; 
    height: 40px; 
}

QCheckBox::indicator:checked {
    image: url(&quot;:/img/icon_light/toggle-button-right.png&quot;);
    width: 40px; 
    height: 40px; 
}


</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="iconSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="autoRepeatDelay">
             <number>0</number>
            </property>
            <property name="autoRepeatInterval">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="fr_lock">
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(250, 250, 250);
border-radius: 15px;
</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QGridLayout" name="gridLayout">
             <property name="leftMargin">
              <number>6</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>3</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <property name="spacing">
              <number>0</number>
             </property>
             <item row="0" column="0">
              <widget class="QCheckBox" name="checkBox_3">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QCheckBox::indicator:unchecked {
    image: url(&quot;:/img/icon_gray/lock-open.png&quot;);
    width: 18px; 
    height:18px;
}

QCheckBox::indicator:checked {
    image: url(&quot;:/img/icon_light/padlock.png&quot;);
    width: 18px; 
    height: 18px;
}


</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="iconSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="autoRepeatDelay">
                <number>0</number>
               </property>
               <property name="autoRepeatInterval">
                <number>0</number>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
